### ISSUE TITLE
Trying to use TwitchRecover2.0a to UNMUTE VOD 

### ISSUE THREAD

**Issue by romelussaladar** [TwitchRecover/TwitchRecover#362]
Hello everyone, I have tried to use TwitchRecover2.0a to unmute the following vod https://vod-secure.twitch.tv/776e2783c157effcd03f_kwissty_41383762859_1665626952/720p60/index-dvr.m3u8  , and it saved the file into a new folder path I created where it saved the new TWITCHRECOVER file. However I can PLAY this M3U8 file in VLC, then when it get's to the SECTIONS I expected to be UNMUTED, the VIDEO image FREEZES, while I can see the VIDEO TIMER continuing as though it was still playing. ALSO there is NO SOUND as it HITS the supposed UNMUTED section, for a short 1 or 2 seconds.

Basically VLC plays all the OTHER sections as normal, UNTIL it gets to the UNMUTED section, then the image freezes, while time counter runs as normal, but the whole video is frozen with NO SOUND.

Can anyone EXPLAIN why the UNMUTED section FREEZES and no sound at the same time ? 

Maybe the TWITCHrecover 2.0a UNMUTE doesn't work?

![image](https://user-images.githubusercontent.com/111546622/196705072-9de60052-850a-488a-8f2d-ec5f31b6c1c7.png)

**Comment by ghost** [comment]
@romelussaladar the unmuted functionality doesnt actually 'unmute' the vod... it allows the vod to be played. If you tried to play the muted segments in the browser without unmuting it.. the vod would buffer forever and would never start.

It would be quite difficult to unmute a video file..(I don't think its possible actually) considering the point of muting is to disable the sound.

**Comment by romelussaladar** [comment]
> @romelussaladar the unmuted functionality doesnt actually 'unmute' the vod... it allows the vod to be played. If you tried to play the muted segments in the browser without unmuting it.. the vod would buffer forever and would never start.
> 
> It would be quite difficult to unmute a video file..(I don't think its possible actually) considering the point of muting is to disable the sound.

Oh ok ok ... cool... yes I understand now !!!

I always noticed when watching the MUTED sections on the browser(via twitch website), then later on getting the M3U8 link to watch on VLC player, yes the video would suddenly STOP then GO BACK to the BEGINNING, whenever it played into the muted sections. I would have to click FORWARD a few minutes to CONTINUE to play the VOD on VLC.

@ItIckeYd  many thanks !!
